cypress/integration/api-information/deleteApplicationsConsumingApi.spec.js

```diff
@@ -33,10 +35,25 @@ describe('Delete an application from an API', () => {
         var applicationName = this.apiData.applications[0].name;
         cy.contains(applicationName).parent().parent().find(".delete-link").click();
         cy.contains("Yes, remove").click();
-        
+
         cy.get('.lbh-page-announcement').contains("Deletion successful!").should('be.visible');
         cy.get('.govuk-summary-list__key').contains(applicationName).should("not.exist");
     })
 
-    // TODO: Add test for API Call here
-})
\ No newline at end of file
+    it('Deleting an application calls the DELETE endpoint', function() {
+        var applicationName = this.apiData.applications[0].name;
+        cy.contains(applicationName).parent().parent().find(".delete-link").click();
+        cy.contains("Yes, remove").click();
+
+        cy.get('@deleteApplication.all').should('have.length', 1);
+    })
+
+    it('Displays an error message if the DELETE endpoint fails', function() {
+        var applicationName = this.apiData.applications[0].name;
+        cy.contains(applicationName).parent().parent().find(".delete-link").click();
+        cy.intercept({method: 'DELETE', url: /api\/v1/gm}, {statusCode: 500}).as("failedDeleteApplication");
+        cy.contains("Yes, remove").click();
+        cy.get('.lbh-dialog__close').click();
+        cy.get('.govuk-error-summary').should('exist');
+    })
```

Comment:
Add another test to ensure that only users in the authorized google group have access to the delete functionality.

Reply:
There is a test in the viewApiInformation file that checks the Delete button is not visible without the google group ("Hide Edit & Delete application buttons if user is not in the allowed groups"). Do I need another one here as well?

Reply:
Oh yes! That's fine then you don't need to add another test :)